pkgs/top-level/all-packages.nix

```diff
@@ -17864,7 +17864,7 @@ with pkgs;
 
   hwloc = callPackage ../development/libraries/hwloc {};
 
-  hydra_unstable = callPackage ../development/tools/misc/hydra/unstable.nix { nix = nixVersions.nix_2_6; };
+  hydra_unstable = callPackage ../development/tools/misc/hydra/unstable.nix { };
```

Comment:
I think it still makes sense to keep pinning nix, because sometimes bindings can break due to updates in nix.

Reply:
Why was the pin moved from v2.8 to v2.6?

Reply:
> Note that the latest version of hydra in this patchset doesn't work with nix_2_4 or nix_2_8 (test failures), so I picked nix_2_6 

It was explained in the prior pr.